bundle/config/mutator/process_target_mode_test.go

```diff
@@ -329,3 +370,90 @@ func TestDisableLockingDisabled(t *testing.T) {
 	require.Nil(t, err)
 	assert.True(t, b.Config.Bundle.Deployment.Lock.IsEnabled(), "Deployment lock should remain enabled in development mode when explicitly enabled")
 }
+
+func TestPrefixAlreadySet(t *testing.T) {
+	b := mockBundle(config.Development)
+	b.Config.Transform.Prefix = "custom_lennart_deploy_"
+
+	m := bundle.Seq(ProcessTargetMode(), ApplyTransforms())
+	diags := bundle.Apply(context.Background(), b, m)
+	require.NoError(t, diags.Error())
+
+	assert.Equal(t, "custom_lennart_deploy_job1", b.Config.Resources.Jobs["job1"].Name)
```

Comment:
As a test local to the process target mode mutator you don't need to run the apply transforms. Can limit to checking that the transform prefix isn't overwritten.

Reply:
That's fine, but it does add coupling.

You get a higher fidelity test by having one at `bundle/tests` that loads the YAML.

No need to change.